aces/Loris
# [instrument_builder] Loading existing instrument corrupts Date element

## Brief summary of changes

When loading an existing instrument and resaving the linst into a new file, the Date element's status field is lost and "undefined" is concatenated to the end of the date element line. This PR fixes the foregoing problem.

#### Testing instructions (if applicable)

1. Go to 'Instrument Builder module', Build tab and build an instrument with date elements
2. Click on 'Save' tab and save the instrument.
3. Read the linst file and see the output.
4. Refresh the instrument builder, and go to the 'Load' tab. Load the instrument linst file that you just created.
5. You can add extra fields to the build if you like, but without editing the date fields, resave the instrument.
6. Read the linst file and notice no changes made to the otherwise untouched file.

#### Link(s) to related issue(s)

* Resolves #6805
